index.html

```diff
@@ -255,12 +255,12 @@ <h3 class="question__title">Chyba s textContent</h3>
           <h3 class="question__title">Komponenta FridgeItem</h3>
           <div class="question__text">
             <p>
-              Vytvořte v čistém JavaScriptu komponentu <code>FridgeItem</code>,
-              která jako vstupní parametr přijímá objekt <code>props</code> s
+              Vytvořte v JavaScriptu komponentu <code>FridgeItem</code>,
```

Comment:
Tady bych doplnil „Vytvořte v JavaScriptu **a JSX**“. Nebo možná rovnou „napište JSX komponentu“, jako je to v následující úloze.